rmf_demos_panel/rmf_demos_panel/gui_server.py

```diff
@@ -37,7 +37,6 @@ def download_webpack():
     # it is advice to update the version (in '.github/workflows/gh_page.yml')
     # whenever a new feature is introduced
     source_url = "https://open-rmf.github.io/rmf_demos/v1.0.0/app.bundle.js"
-
```

Comment:
Nit, you can just do git revert to revert changes to make sure the file doesn't appear edited when we merge, even with a single line it will pollute history. Ideally the whole PR would only have 4 files changed